sos/plugins/__init__.py

```diff
@@ -180,6 +180,40 @@ def is_installed(self, package_name):
         '''Is the package $package_name installed?'''
         return self.policy().pkg_by_name(package_name) is not None
 
+    def do_cmd_private_sub(self, cmd):
+        '''Remove certificate and key output archived by sosreport. cmd
+        is the command name from which output is collected (i.e. exlcuding
```

Comment:
Just a typo "exlcuding" (I wont be such grammar nazi if I wont have another comment :) ).